src/wallet/rpcwallet.cpp

```diff
@@ -5964,6 +5994,7 @@ static const CRPCCommand commands[] =
     { "wallet",             "dumpprivkey",              &dumpprivkey,              true  },
     { "hidden",             "dumpwallet",               &dumpwallet,               true  },
     { "wallet",             "encryptwallet",            &encryptwallet,            true  },
+    { "wallet",             "converttex",               &converttex,               true  },
```

Comment:
I mildly object to `converttex`, as we've generally used a `z_` prefix for new methods (to avoid collisions with Bitcoin Core-derived RPCs). I only mildly object because we've not been completely consistent here (usually only however when we've needed to add a new method replacing one of our own earlier `z_`-prefixed methods though, so not the same as here).
```suggestion
    { "wallet",             "z_converttex",               &converttex,               true  },
```

Reply:
The reason I chose `converttex` is that it's a transparent-only function, but I don't feel strongly about it.